<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_options</class>
 <widget class="QDialog" name="C_options">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>492</width>
    <height>707</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>492</width>
    <height>707</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>492</width>
    <height>707</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="error_list">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>470</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Modification adresse et port </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>530</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Base de données SECU</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>0</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Paramètres généraux</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>315</y>
     <width>68</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Code Parental  actuel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_lieuStockage">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>420</y>
     <width>232</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_nCode">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>360</y>
     <width>232</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>43</y>
     <width>48</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Utilisateur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_tempo">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>210</y>
     <width>232</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>69</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Nouveau code parental</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Lieu de Stockage</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_code">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>315</y>
     <width>232</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>4</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_fixe">
   <property name="geometry">
    <rect>
     <x>127</x>
     <y>247</y>
     <width>232</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cheminTempo">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>210</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Sélection</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>64</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>471</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Contrôle parental</string>
   </property>
   <widget class="QPushButton" name="btn_enr_CP">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chemin  tempo</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Chemin fixe</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>471</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Lieux de stockage</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>471</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Dossiers d'enregistrement</string>
   </property>
   <widget class="QPushButton" name="pushButton_cheminFixe">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>50</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Sélection</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>161</height>
    </rect>
   </property>
   <property name="title">
    <string>Base de données DVDFLIX</string>
   </property>
   <widget class="QPushButton" name="btn_enr_BD_dvd">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>150</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string>Enregistrer</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_adresse">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>232</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_port">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>110</y>
      <width>232</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_mdp">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>232</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="txt_utilisateur">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>232</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>39</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>20</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_modifier">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>650</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>150</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.482412, y1:0.983, x2:0.493025, y2:0.239, stop:0 rgba(85, 170, 255, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 85, 255);
border-radius: 10px;
border-width: 1px;
border-style: solid;</string>
   </property>
   <property name="text">
    <string>Modifier</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>624</y>
     <width>39</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Port DB </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_portDbSecu">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>624</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_adrDbSecu">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>580</y>
     <width>241</width>
     <height>18</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(0, 85, 255);
border-radius:15px;
border-width: 1px;
border-style: solid;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>580</y>
     <width>58</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse DB </string>
   </property>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>error_list</zorder>
  <zorder>label_9</zorder>
  <zorder>label_6</zorder>
  <zorder>line</zorder>
  <zorder>label_10</zorder>
  <zorder>pushButton</zorder>
  <zorder>label_5</zorder>
  <zorder>txt_lieuStockage</zorder>
  <zorder>txt_nCode</zorder>
  <zorder>label</zorder>
  <zorder>txt_tempo</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>txt_code</zorder>
  <zorder>txt_fixe</zorder>
  <zorder>pushButton_cheminTempo</zorder>
  <zorder>label_2</zorder>
  <zorder>label_13</zorder>
  <zorder>label_14</zorder>
  <zorder>pushButton_modifier</zorder>
  <zorder>label_8</zorder>
  <zorder>txt_portDbSecu</zorder>
  <zorder>txt_adrDbSecu</zorder>
  <zorder>label_7</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
